examples/with-react-md/static/react-md.light_blue-yellow.min.css

```diff
@@ -0,0 +1 @@
+../node_modules/react-md/dist/react-md.light_blue-yellow.min.css
```

Comment:
as I understand, to create it I need to `cd` into `static` and then run `ln -s ../node_modules/react-md/dist/react-md.light_blue-yellow.min.css react-md.light_blue-yellow.min.css` ?